File: Seccion 11 - Funciones en C++/Seccion_11_Dart/05_ejercicio_03.dart

import 'dart:io';

int numero, exponente;

void pedirDatos() {
  stdout.writeln('Digite el número: ');
  numero = int.parse(stdin.readLineSync());
  stdout.writeln('Digite el exponente: ');
  exponente = int.parse(stdin.readLineSync());
}

void funpot(int x, int y) {
  int resultado = 1;

  for (int i = 1; i <= y; i++) {
    resultado *= x;
  }

  print('El resultado de $x elevado a $y es: $resultado');
}

void main() {
  pedirDatos();
  funpot(numero, exponente);
}

File: Seccion 11 - Funciones en C++/Seccion_11_Python/05_ejercicio_03.py

numero = 1
exponente = 1

def pedir_datos():
    global numero, exponente
    numero = int(input('Digite el número: '))
    exponente = int(input('Digite el exponente: '))

def funpot(x, y):
    resultado = 1

    for i in range(y):
        resultado *= x;
    
    print('El resultado de {} elevado a {} es: {}'.format(x, y, resultado))

pedir_datos()
funpot(numero, exponente)